app/models/item.rb

```diff
@@ -1,5 +1,18 @@
 class Item < ActiveRecord::Base
   belongs_to :user
 
-  attr_accessor :task
+  attr_accessor :task, :time_remaining
+
+  def time_remaining(entry)
```

Comment:
it looks like `entry` here is the item in question, right?  if so, that item is already the receiver of this instance method and you can invoke the receiver of the method through the keyword `self`.
and when you want to invoke the method (in your show view, for example), you would then call `time_remaining` on a given item:  `item_variable.time_remaining` (where item_variable is whatever variable you're using to hold the item object in memory)